lambda_function.py

```diff
@@ -31,3 +30,54 @@ def lambda_handler(event, context):
     put_order_success = dynamo_client.put_order(order, cust)
     if not put_order_success:
         print("Failed to save new order:" + order)
+    
+    print(order)
+    esim_client = EsimGoClient()
+    esim_order_details = esim_client.new_order(order1)
+
+    if not esim_order_details:
+        print("Failed to generate new order in esimGo:" + str(order1))
+        return
+    
+
+    order_id = dynamo_client.put_esim_order(esim_order_details, order1)
+    if not order_id:
+        print("Failed to generate new order:" + esim_order_details)
+        return
+    else:
+        print(order_id)
+
+  
+    esim_order_details = esim_client.get_esim_details(order_id)
+
+    if not esim_order_details:
+        print("Failed to get esim details:" + esim_order_details)
+        return
+    else:
+        response = dynamo_client.update_esim_order(order_id, esim_order_details)
+    
+    image_data = esim_client.get_esim_qrcode(order_id)
+
+    if not image_data:
+        print("Failed to get esim details:" + esim_order_details)
+        return
+    else:
+        response = dynamo_client.update_esim_qr_code(order_id, image_data)
+    
+    
+    qr_codes = dynamo_client.get_qr_code_from_db(order_id)
+    if not qr_codes:
+        print("Failed to get qr code from db :" + qr_codes)
+        return
+    
+
+    esim_details = dynamo_client.get_esim_from_db(order_id)
+    if not esim_details:
+        print("Failed to get esim_details from db :" + esim_details)
+        return
+  
+
+    if qr_codes:
+        email_client.send_email_with_qr_code('sivasankar.selva@gmail.com', 'QR Code Email', qr_codes, esim_details)
```

Comment:
have we got a ticket to swap this out when we need to?

Reply:
@zaine2003 you mean the email address right? testing I added we can change to customer email here.